mappings/aaf267c77f09a5b4f006009a3235f3a9765660ccf2e6e1e0380303d54b65726d697420746865204675636b.json

```diff
@@ -0,0 +1,64 @@
+{
+    "subject": "aaf267c77f09a5b4f006009a3235f3a9765660ccf2e6e1e0380303d54b65726d697420746865204675636b",
+    "url": {
+        "sequenceNumber": 0,
+        "value": "https://kermitthefuck.com",
```

Comment:
Hi @mrsqurriel,

if your site is still not up, you can update this value, once it is running. The URL value is not required for the submission. For now, we can't verify the URL. Please, update the PR without URL value or with a valid page.

Thanks!